cmd/all-in-one/Dockerfile

```diff
@@ -15,6 +15,9 @@ EXPOSE 5778
 # Collector HTTP
 EXPOSE 14268
 
+# Collector gRPC
+EXPOSE 14270
```

Comment:
+1 it has to be changed. Do we have any preference?

Reply:
I don't care which, as a long as it's unique.

We my want to move all ports into common package across all components so that it's easier to check for uniqueness.

Reply:
I have used 14250